packages/blade/src/components/ListView/ListViewFilters.web.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react';
+import type { ListViewFilterProps } from './types';
+import { ListViewFiltersProvider } from './ListViewFiltersContext.web';
+import { makeAnalyticsAttribute } from '~utils/makeAnalyticsAttribute';
+import BaseBox from '~components/Box/BaseBox';
+import { MetaConstants, metaAttribute } from '~utils/metaAttribute';
+import { FilterIcon } from '~components/Icons';
+import { Button } from '~components/Button';
+import { Counter } from '~components/Counter';
+import { Box } from '~components/Box';
+
+const ListViewFilters = ({
+  testID,
+  children,
+  quickFilters,
+  onSearchChange,
+  ...rest
+}: ListViewFilterProps): React.ReactElement => {
+  const [showFilters, setShowFilters] = useState(false);
+  const [selectedFilters, setSelectedFilters] = useState<string[]>([]);
+  const [clearFiltersCallbackTriggerer, setClearFiltersCallbackTriggerer] = useState<number>(0);
```

Comment:
This name doesn't make sense it's called a "callback trigerer" but the value is a number?